src/position/template/app-token.template.position-fetcher.ts

```diff
@@ -303,4 +304,35 @@ export abstract class AppTokenTemplatePositionFetcher<
 
     return balances as AppTokenPositionBalance<V>[];
   }
+
+  async getRawBalances(address: string): Promise<RawAppTokenBalance[]> {
+    const multicall = this.appToolkit.getMulticall(this.network);
+    const appTokens = await this.appToolkit.getAppTokenPositions({
+      appId: this.appId,
+      network: this.network,
+      groupIds: [this.groupId],
+    });
+
+    return Promise.all(
+      appTokens.map(async appToken => ({
+        key: this.appToolkit.getPositionKey(appToken),
+        balance: (await this.getBalancePerToken({ multicall, address, appToken })).toString(),
+      })),
+    );
+  }
+
+  async drillRawBalances(balances: RawAppTokenBalance[]): Promise<AppTokenPositionBalance<V>[]> {
```

Comment:
Not a huge fan of the method name here, i'm open to suggestions for a different name